src/Misc/layoutbin/en-US/strings.json

```diff
@@ -58,6 +58,165 @@
+    "For unconfigure help, see: .{0}config.{1} remove --help",
+    "",
+    "{2}",
+    "",
+    "CLI-WIDTH-OPTIONS-(35-CHARS)-------CLI-WIDTH-DESCRIPTION-(70-CHARS)--------------------------------------",
+    "Configure options:",
+    " --pool <pool>                     Pool name for the agent to join",
+    " --agent <agent>                   Agent name",
+    " --replace                         Replace the agent in a pool. If another agent is listening by that",
+    "                                   name, it will start failing with a conflict.",
+    " --work <workDirectory>            Work directory where job data is stored. Defaults to _work under the",
+    "                                   root of the agent directory. The work directory is owned by a given",
+    "                                   agent and should not share between multiple agents.",
+    " --acceptTeeEula                   macOS and Linux only. Accept the TEE end user license agreement.",
+    "",
+    "CLI-WIDTH-OPTIONS-(35-CHARS)-------CLI-WIDTH-DESCRIPTION-(70-CHARS)--------------------------------------",
+    "Startup options (Windows only):",
+    " --runAsService                    Configure the agent to run as a Windows service. Requires",
+    "                                   administrator permission.",
+    " --runAsAutoLogon                  Configure auto logon and run the agent on startup. Requires",
+    "                                   administrator permission.",
+    " --windowsLogonAccount <account>   Used with --runAsService or --runAsAutoLogon. Specify the Windows user",
+    "                                   name in the format: domain\\userName or userName@domain.com",
+    " --windowsLogonPassword <password> Used with --runAsService or --runAsAutoLogon. Windows logon password.",
+    " --overwriteAutoLogon              Used with --runAsAutoLogon. Overwrite any existing auto logon on the",
+    "                                   machine.",
+    " --noRestart                       Used with --runAsAutoLogon. Do not restart after configuration",
+    "                                   completes.",
+    "",
+    "CLI-WIDTH-OPTIONS-(35-CHARS)-------CLI-WIDTH-DESCRIPTION-(70-CHARS)--------------------------------------",
+    "Deployment group options:",
+    " --deploymentGroup                 Configure the agent as a deployment group agent.",
+    " --projectName <name>              Used with --deploymentGroup. Team project name.",
+    " --addDeploymentGroupTags          Used with --deploymentGroup. Specify to add deployment group tags.",
+    " --deploymentGroupName <anme>      Used with --deploymentGroup. Deployment group for the agent to join.",
+    " --deploymentGroupTags <tags>      Used with --addDeploymentGroupTags. A comma separated list of tags for",
+    "                                   the deployment group agent. For example \"tag1,tag2\".",
+    "",
+    "{3}",
+    "",
+    "CLI-WIDTH-UNLIMITED-FOR-EXAMPLES-----",
```

Comment:
I'm thinking about not printing the examples by default, unless the user specifies `./config.sh --help --examples`. When the help is always printed, `./config.sh --help` is a bad experience. A bunch of stuff is dumped to the console and the immediate action is start scrolling backwards through a bunch of dense text. It's hard to quickly sift through and try to understand what your eye should start focusing on.